tests/suites/test_suite_psa_crypto_driver_wrappers.function

```diff
@@ -2567,3 +2975,302 @@ exit:
     PSA_DONE();
 }
 /* END_CASE */
+
+/* BEGIN_CASE depends_on:PSA_WANT_ALG_JPAKE:PSA_WANT_KEY_TYPE_ECC_KEY_PAIR:PSA_WANT_ECC_SECP_R1_256:PSA_WANT_ALG_SHA_256 */
```

Comment:
```suggestion
/* BEGIN_CASE depends_on:PSA_WANT_ALG_JPAKE */
```
Actually I think we should only have the PSA_WANT_ALG_JPAKE dependency here. If the PSA configuration is enabled and PSA_WANT_ALG_JPAKE is defined then all the MBEDTLS_... dependencies are enforced by config_psa.h. If the PSA configuration is not enabled, if MBEDTLS_ECJPAKE_C is enabled PSA_WANT_ALG_JPAKE is defined by config_psa.h and the dependencies on the other MBEDTLS_... are enforced by check_config.h. Sorry for the noise.